frontend-react/src/components/DataDashboard/ReportDetails/ReportDetailsTable.module.scss

```diff
@@ -0,0 +1,37 @@
+.ReportDetailsTable {
+    padding: 32px 33px;
+    background-color: #F1F3F6;
```

Comment:
Is there a color token we can use?

Reply:
@etanb this color is not part of the uswds themes

Reply:
Does `gray-cool-4` from https://designsystem.digital.gov/design-tokens/color/system-tokens/ work?